src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -188,40 +198,56 @@ export default {
                 image.selected = true;
             }
         },
-        stopListening() {
-            if (this.unsubscribe) {
-                this.unsubscribe();
-                delete this.unsubscribe;
+        boundsChange(bounds, isTick) {
+            this.bounds = bounds;
+            this.requestHistory(isTick);
+        },
+        requestHistory(isTick) {
+            if(!isTick) {
```

Comment:
As with LAD tables, I think this check would be better in `boundsChange()`

Reply:
Agreed, moved it out.